apps/web/src/app/dashboard/[name]/create-project/page.tsx

```diff
@@ -0,0 +1,25 @@
+import VideUploadForm from "~/components/create-project/create-project-form";
+import { createEndpointForMuxUpload } from "~/server/actions/mux";
+import { getOwnOrganizationByName } from "~/server/actions/organization";
+
+type CreateProjectPageProps = {
+  params: {
+    name: string;
+  };
+};
+
+export default async function CreateProjectPage({
+  params,
+}: CreateProjectPageProps) {
+  const [upload, uploadErr] = await createEndpointForMuxUpload();
+  if (uploadErr !== null) {
+    throw new Error(uploadErr);
```

Comment:
Add error handling, that's consistent with the rest of the app. Toast the error to the user.

Reply:
Exactly

Reply:
This is a server component. It's already handled by next router which will redirect to `error.tsx` and display the error message.